packages/nimble-components/src/table/components/row/template.ts

```diff
@@ -30,6 +30,7 @@ export const template = html<TableRow>`
                         :cellState="${x => x.cellState}"
                         :cellViewTemplate="${x => x.column.columnInternals.cellViewTemplate}"
                         :column="${x => x.column}"
+                        :recordId="${(_x, c) => c.parent.recordId}"
```

Comment:
I believe we generally use `_` rather than `_variableName` when something is unused -- at least that's what we use in this file.
```suggestion
                        :recordId="${(_, c) => c.parent.recordId}"
```